sts-mock/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "one-login-sts-mock",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "format": "./node_modules/.bin/prettier --write '**/*.ts'",
+    "format:check": "./node_modules/.bin/prettier --check '**/*.ts'",
+    "lint": "./node_modules/.bin/eslint ./src",
+    "test:infra": "./node_modules/.bin/jest --testPathPattern=infrastructure-tests/",
+    "test:unit": "./node_modules/.bin/jest --detectOpenHandles",
+    "ci": "npm run test:unit && npm run format:check && npm run lint"
+  },
+  "engines": {
+    "node": "20"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.8.0",
+    "@types/aws-lambda": "8.10.142",
+    "@types/jest": "29.5.12",
+    "@types/js-yaml": "^4.0.9",
+    "eslint": "^9.8.0",
+    "jest": "29.7.0",
+    "jest-junit": "16.0.0",
+    "prettier": "3.3.3",
+    "ts-jest": "29.2.4",
+    "ts-node": "10.9.2",
+    "typescript": "^5.5.4",
+    "typescript-eslint": "^8.0.1",
+    "yaml-cfn": "^0.3.2"
+  },
+  "dependencies": {
+    "@aws-lambda-powertools/logger": "^2.6.0",
+    "@types/node": "^20.14.14",
+    "aws-cdk-lib": "^2.151.0",
```

Comment:
q: _if_ this is only used for infrastructure tests - should this be a devDependency? I can't remember whether it's a sam dependency